air-script/tests/main.rs

```diff
@@ -1,191 +1,327 @@
-        .transpile()
+    let generated_winterfell = Test::new("tests/variables/variables.air".to_string())
+        .transpile(Target::Winterfell)
         .unwrap();
 
     let expected = expect_file!["variables/variables.rs"];
-    expected.assert_eq(&generated_air);
+    expected.assert_eq(&generated_winterfell);
+
+    let generated_masm = Test::new("tests/variables/variables.air".to_string())
+        .transpile(Target::Masm)
+        .unwrap();
+
+    let expected = expect_file!["variables/variables.masm"];
+    expected.assert_eq(&generated_masm);
 }
 
 #[test]
 fn trace_col_groups() {
-    let generated_air = Test::new("tests/trace_col_groups/trace_col_groups.air".to_string())
-        .transpile()
+    let generated_winterfell = Test::new("tests/trace_col_groups/trace_col_groups.air".to_string())
+        .transpile(Target::Winterfell)
         .unwrap();
 
     let expected = expect_file!["trace_col_groups/trace_col_groups.rs"];
-    expected.assert_eq(&generated_air);
+    expected.assert_eq(&generated_winterfell);
+
+    let generated_masm = Test::new("tests/trace_col_groups/trace_col_groups.air".to_string())
+        .transpile(Target::Masm)
+        .unwrap();
+
+    let expected = expect_file!["trace_col_groups/trace_col_groups.masm"];
+    expected.assert_eq(&generated_masm);
 }
 
 #[test]
 fn indexed_trace_access() {
-    let generated_air =
+    let generated_winterfell =
         Test::new("tests/indexed_trace_access/indexed_trace_access.air".to_string())
-            .transpile()
+            .transpile(Target::Winterfell)
             .unwrap();
 
     let expected = expect_file!["indexed_trace_access/indexed_trace_access.rs"];
-    expected.assert_eq(&generated_air);
+    expected.assert_eq(&generated_winterfell);
+
+    let generated_masm =
+        Test::new("tests/indexed_trace_access/indexed_trace_access.air".to_string())
+            .transpile(Target::Masm)
+            .unwrap();
+
+    let expected = expect_file!["indexed_trace_access/indexed_trace_access.masm"];
+    expected.assert_eq(&generated_masm);
 }
 
 #[test]
 fn random_values() {
-    let generated_air = Test::new("tests/random_values/random_values_simple.air".to_string())
-        .transpile()
-        .unwrap();
+    let generated_winterfell =
+        Test::new("tests/random_values/random_values_simple.air".to_string())
+            .transpile(Target::Winterfell)
+            .unwrap();
 
     let expected = expect_file!["random_values/random_values.rs"];
-    expected.assert_eq(&generated_air);
+    expected.assert_eq(&generated_winterfell);
 
-    let generated_air = Test::new("tests/random_values/random_values_bindings.air".to_string())
-        .transpile()
-        .unwrap();
+    // let generated_masm = Test::new("tests/random_values/random_values_simple.air".to_string())
+    //     .transpile(Target::Masm)
+    //     .unwrap();
+    // let expected = expect_file!["random_values/random_values.masm"];
+    // expected.assert_eq(&generated_masm);
```

Comment:
nit: I think it would be good to add a `TODO` comment about the non-determinism so it's clear why this is commented out. Same thing in the other affected tests below

Reply:
added